src/store/index.js

```diff
@@ -753,6 +753,11 @@ export class ApplicationStore {
     resetSubGraphLayout() {
         this.subGraphLayout = {};
     }
+
+    @action
+    resetZoom() {
+        this.zoom = -2.3;
```

Comment:
isn't the default value for the zoom stored in the dataset defaults? at least that is what I remembered

Reply:
Good catch, thanks. See 0d5f0eb.